sources/NotoSansMath-Regular v2-500.glyphspackage/fontinfo.plist

```diff
@@ -0,0 +1,7074 @@
+sub similarAboveLessThanAboveEqualsSign by similarAboveLessThanAboveEqualsSign.rtlm;
+sub similarMinusSimilar by similarMinusSimilar.rtlm;
+sub similarOrGreaterThan by similarOrGreaterThan.rtlm;
+sub similarOrLessThan by similarOrLessThan.rtlm;
+sub sinewave by sinewave.rtlm;
+sub slopingLargeAnd by slopingLargeAnd.rtlm;
+sub slopingLargeOr by slopingLargeOr.rtlm;
+sub solidusWithOverbar by solidusWithOverbar.rtlm;
+sub sphericalAngleOpeningLeft by sphericalAngleOpeningLeft.rtlm;
+sub sphericalangle by sphericalangle.rtlm;
+sub subsetOfAboveAlmostEqualTo by subsetOfAboveAlmostEqualTo.rtlm;
+sub subsetOfAboveNotEqualTo by subsetOfAboveNotEqualTo.rtlm;
+sub subsetOfAboveTildeOperator by subsetOfAboveTildeOperator.rtlm;
+sub succeedsAboveAlmostEqualTo by succeedsAboveAlmostEqualTo.rtlm;
+sub succeedsAboveNotAlmostEqualTo by succeedsAboveNotAlmostEqualTo.rtlm;
+sub succeedsAboveNotEqualTo by succeedsAboveNotEqualTo.rtlm;
+sub succeedsAboveSingleLineNotEqualTo by succeedsAboveSingleLineNotEqualTo.rtlm;
+sub summation by summation.rtlm;
+sub summationWithIntegral by summationWithIntegral.rtlm;
+sub summationdoubleStruck by summationdoubleStruck.rtlm;
+sub supersetOfAboveAlmostEqualTo by supersetOfAboveAlmostEqualTo.rtlm;
+sub supersetOfAboveNotEqualTo by supersetOfAboveNotEqualTo.rtlm;
+sub supersetOfAboveTildeOperator by supersetOfAboveTildeOperator.rtlm;
+sub surfaceintegral by surfaceintegral.rtlm;
+sub theredoesnotexist by theredoesnotexist.rtlm;
+sub threeDimensionalAngle by threeDimensionalAngle.rtlm;
+sub tildeOperatorWithDotAbove by tildeOperatorWithDotAbove.rtlm;
+sub tildeOperatorWithRisingDots by tildeOperatorWithRisingDots.rtlm;
+sub tripleSolidusBinaryRelation by tripleSolidusBinaryRelation.rtlm;
+sub tripleintegral by tripleintegral.rtlm;
+sub tripletilde by tripletilde.rtlm;
+sub tripleverticalbarrightturnstile by tripleverticalbarrightturnstile.rtlm;
+sub turnedAngle by turnedAngle.rtlm;
+sub twoJoinedSquares by twoJoinedSquares.rtlm;
+sub partialdiffbold-math by partialdiffbold-math.rtlm;
+sub partialdiffitalic-math by partialdiffitalic-math.rtlm;
+sub partialdiffbolditalic-math by partialdiffbolditalic-math.rtlm;
+sub partialdiffsansbold-math by partialdiffsansbold-math.rtlm;
+sub partialdiffsansbolditalic-math by partialdiffsansbolditalic-math.rtlm;
+sub upperLeftCornerWithDot by upperLeftCornerWithDot.rtlm;
+sub verticalBarTripleRightTurnstile by verticalBarTripleRightTurnstile.rtlm;
+sub volumeintegral by volumeintegral.rtlm;
+sub wreathproduct by wreathproduct.rtlm;
+sub zNotationRelationalComposition by zNotationRelationalComposition.rtlm;
+sub zNotationSchemaComposition by zNotationSchemaComposition.rtlm;
+sub zNotationSchemaPiping by zNotationSchemaPiping.rtlm;
+sub zNotationSchemaProjection by zNotationSchemaProjection.rtlm;
+sub znotationbagmembership by znotationbagmembership.rtlm;
+";
+tag = rtlm;
+},
+{
+code = "# FIXME:TODO:REVIEW: for completeness toward end of design -- make sure to update classes to match
```

Comment:
Is there still an action item here?

Reply:
I think there might be - some script math letters are not covered by these classes, as you look for `script-math` at the end of the glyph name, but the glyph names don't seem to be consistent:

```
Ascript-math
Bscript
Cscript-math
Dscript-math
Escript
Fscript
Gscript-math
```

Reply:
I see you fixed that in commit b76726db. The difference in naming is historical Unicode naming, since things like `Bscript` and `Escript` existed before the `-math` variants.